### PR TITLE
Clean up component endpoint.

### PR DESCRIPTION
**Pull request by gislenius**
This reverts commit 4b71137e2591a3c6c79f691dde382a47cb1613c6.

### CODE REVIEW

**Comment by gislenius** [comment]
http://secxbrl-dev6.28.io/v1/_queries/public/api/components.jq?token=foobar&ticker=ko&fi
scalYear=2013&fiscalPeriod=FY&format=html

File: documentation/4-components-3-metadata.md [hunk 1]
```diff
@@ -16,3 +19,15 @@ Japan-specific metadata
 
 In the Japanese profile, there is also:
 - *Disclosure*: The disclosure with which the component was stamped, if available.
+
+SEC-specific metadata
+-----------------------
+
+In the SEC profile, there is also:
+- *Disclosure*: The disclosure with which the component was stamped, if available.
```
Comment (dknochen) [inline documentation/4-components-3-metadata.md]: we could list the disclosures here if we cannot add it here: https://github.com/28msec/cellstore/pull/938

**Comment by dknochen** [comment]
tests fail

File: documentation/4-components-3-metadata.md (hunk 1 above)
Comment (gislenius) [inline documentation/4-components-3-metadata.md]: Done!

**Comment by gislenius** [comment]
@dknochen tests pass ;-)

**Pull request merged**